### DISCUSSION TITLE
Polska zgoda ciasteczkowa - czy nadal rozwijana? Czy EOL?

### DISCUSSION THREAD

**Discussion by seba2282** [MajkiIT/polish-ads-filter#21958] [General]
Tak jak w temacie. Czy nadal jest to aktywny projekt, czy został porzucony? Widziałbym potencjał jako alternatywa dla I don't care about cookies właśnie w niej. 😁😁😁

**Comment by hawkeye116477** [comment]
Listy są dostępne na https://github.com/FiltersHeroes/PCCassets, parę tygodni temu była aktualizacja. Jak chcesz, to możesz zająć się listą EU, bo brakuje chętnych. Chyba, że kiedyś wykombinujemy, by bot przerabiał listę z IDCAC.
Natomiast co do samego rozszerzenia, nowych funkcji na razie nie dodaję, choć są [plany](https://github.com/orgs/FiltersHeroes/projects/1/views/1), ale nie wiem czy coś z tego będzie, zobaczymy. Jak chcesz, to złóż petycję, jak się podpisze ponad 100 osób to wtedy może rozważę :smile:.
A na razie na liście priorytetowej mam 3 projekty do ogarnięcia - TSIŚ, CEDFF i PLP.

**Comment by seba2282** [comment]
> TSIŚ, CEDFF i PLP.

A możesz coś więcej o tym powiedzieć? Do czego to?

**Reply by hawkeye116477** [reply]
W zasadzie pewnie można się tego domyśleć po historii mojej aktywności GH, ale opowiem.

CEDFF =>Check Expired Domains From Filterlists
PLP => playnite-legendary-plugin
TSIŚ => testowanie symetrycznego internetu światłowodowego, czyli muszę sprawdzić czy mnie nie oszukują i mam prawdziwy FTTH PON do 200/200 od tych, co niby Orange często na nich psioczy, co nie @krystian3w :rofl:. Przy okazji dodam, że poprzednio miałem tylko radio-linię 20/2 za 10 zł drożej, bo jedyna alternatywa to był Orange z obowiązkową opłatą dodatkową za linię telefoniczną i nawet nadal to mają na stronie dla mnie, bo ŚI niby buduje, ale planują dopiero na Sylwestra za rok ukończyć prace/świadczyć usługi na mojej ulicy, pomimo, że jako pierwsi na mapce się pojawili. Tak więc wychodzi na to, że jest tak jak to powiadają - „ostatni będą pierwszymi".

**Reply by seba2282** [reply]
Dobre 🤣
Teraz jestem w temacie. I nie zaglądałem wcześniej, to nie wiedziałem, co te skróty znaczą. Tak więc dzięki za wyjaśnienie.

**Reply by hawkeye116477** [reply]
No to przy okazji pochwalę się wynikami.
Kiedyś było tak na radiolinii.
![2023-07-28_17-18](https://github.com/MajkiIT/polish-ads-filter/assets/19818572/219a5b64-ac0c-480c-9602-b8330f38c0c4)

A teraz.
![2023-09-03_20-51](https://github.com/MajkiIT/polish-ads-filter/assets/19818572/3e68bc2d-49ea-4ce7-b42d-b0d1dcddd046)
<img src="https://github.com/MajkiIT/polish-ads-filter/assets/19818572/0316b01d-3cb6-4eea-b6b6-503e3e9920e5" width=40% height=40%>

I jakby ktoś był ciekawy, to nie UPC ani Inea (zarówno jednych, jak i drugich nawet nie ma w moim mieście), tylko taka lokalna kablówka.

**Reply by seba2282** [reply]
No to nieźle 😅

**Comment by krystian3w** [comment]
Część możesz zgłaszać do uAssets jak masz gotowe działające skryptlety:

`trusted-set-cookie`

Jedynie mogą uznać, że za mało znasz JS i nie lubisz np. `set-constant` jaki można odpalać z dowolnego pliku.

**Reply by seba2282** [reply]
I to wystarczy? 
Ja tak się zastanawiam, dlaczego te likwidowanie plików cookies nie jest w pełni zaimplementowane w ublock origin. Wtedy ludzie nie martwili by się o dodatkowe rozszerzenia i to też rozwiązałoby problem z Firefox na android.

**Reply by hawkeye116477** [reply]
> likwidowanie plików cookies

Raczej chyba nam chodzi o komunikaty, a nie same pliki. Niektóre wymagają klikania albo robienia przekierowania, a gorhill nie chce tego dodać, bo to mogłoby być niebezpieczne, ktoś mógł by zrobić listę z przekierowaniami na jakieś wirusy albo klikanie w podobne elementy. Niby mógłby też dać ograniczenie, żeby tylko dla zaufanych list było, ale na te rozwiązanie chyba też nie był zbyt chętny.

**Reply by seba2282** [reply]
To niby jak Firefox miał dostać funkcję usuwania tych informacji o cookies, jeżeli dba o bezpieczeństwo, a mówisz, że gorhill nie chce dodać bo to niebezpieczne ?

**Reply by hawkeye116477** [reply]
Jeżeli sami to kontrolują, to nie ma takiego problemu. W przypadku uBO jest wiele list od różnych autorów.
Nie wiemy też jak te rozwiązanie dla FF miało działać, może bez klikania i przekierowań, tylko np same pieczenie ciasteczek.

**Reply by seba2282** [reply]
No i teraz już zrozumiałem do końca.

**Reply by krystian3w** [reply]
https://github.com/gorhill/uBlock/wiki/Resources-Library#trusted-set-cookiejs-
https://github.com/gorhill/uBlock/wiki/Resources-Library#trusted-set-local-storage-itemjs-

Jedynie muszą być chętni do dokumentowania, że inne metody wycięcia pop-upa psują stronę np. brak wideo, sekcji komentarzy, osadzone tweety i posty z instagrama są pozbawione grafik (czyli często analizy czy ruszy [`set-constant`](https://github.com/gorhill/uBlock/wiki/Resources-Library#set-constantjs-) lub [`aopr`](https://github.com/gorhill/uBlock/wiki/Resources-Library#aoprjs-) nie unikniesz).

Szkoda, że tak późno ktoś udokumentował parametry na wiki.

**Reply by seba2282** [reply]
Dobrze wiedzieć

**Comment by hawkeye116477** [comment]
@seba2282 
Mam dobre wieści dla fanów, po 3 latach będzie wielki powrót/aktualizacja :smile: 
Prawdopodobnie będzie CodeMirror 6, nowe narzędzia do akceptacji, możliwość wyłączenia pojedynczych reguł, IntelliSense, nowy wygląd i mały easter egg.
<img width="751" height="569" alt="2025-09-12_011326" src="https://github.com/user-attachments/assets/effd80f8-e88f-4867-92ce-e650d156e63e" />

**Reply by seba2282** [reply]
Niemożliwe. Rozszerzenie wstało z martwych. ;)

**Reply by hawkeye116477** [reply]
W przypadku Chromium będzie MV3 (w tym akurat przypadku wszystko powinno działać jak do tej pory, czyli nie ma żadnej wersji lite), ale to wkrótce, tzn aż @krystian3w wrzuci i przeanalizują kod, czyli pewnie za miesiąc :smile:.

Ale za to już powinna być nowa wersja dla przeglądarek UXP i liska, czyli możesz już szukać niespodzianek.

**Reply by krystian3w** [reply]
> <img width="1502" height="694" alt="obraz" src="https://github.com/user-attachments/assets/621077e1-0409-4633-bda2-e8b5cf36df67" />


Zostaje już tylko czekać, jak się uprą na aktualizacje całego dodatku (aktualizacje w tle zrobione nie tak, jak zrobił to zespół ABP dla ich trybu różnicowego), to chyba trzeba podpiąć moje konto pod konsolę Google strony netlify, tak by co 30 dni nie odnawiać akredytacji dla np. Circle CI.

**Comment by hawkeye116477** [comment]
A tak a propos ciasteczek, to niby ma być jakaś rewolucja, że będzie tylko zgoda na poziomie ustawień przeglądarki albo systemu wg nowych przepisów, ale chyba dopiero za parę lat. Jak by to tak działało, to by dodatek stał się kompletnie zbędny.

**Reply by krystian3w** [reply]
Wtedy to nawet niektóre listy mogą być zbędne:
- bo momentalnie znika telemetria w Europie, Kalifornii, Teksasie, Brazylii czy RPA
   - chyba że kary za profilowanie bez zgody będą drakońsko niskie lub przegrywać w sądach
- lista ciasteczkowa wycinałaby:
   - paski pamiętające ustawę z 2002-2009,
   - modale RODO ładowane ze przestarzałych skryptów lokalnych/wtyczek (ze zdalnego zasobu pewnie będą przekazywać puste pliki)

Wadą po części będzie np. brak personalizacji gdzie akceptujemy komunikację z socjal mediami (osadzone posty lub wideo), jak na początku będzie model wszędzie albo nigdzie.
